debian/changelog

```diff
@@ -43,6 +43,12 @@ kano-desktop (3.10.0-1) unstable; urgency=low
 
  -- Team Kano <dev@kano.me>  Wed, 15 Mar 2017 12:10:00 +0000
 
+ kano-desktop (3.9.2-0) unstable; urgency=low
```

Comment:
Hmm, Haven't inserted a version into the middle of a changelog before. i guess it should work....

Reply:
Done it before, dr will build the latest version specified in the changelog anyway so as long as the changelog is valid syntactically it'll work :)